docs/design-proposals/apiserver/Action-API-design.md

```diff
@@ -94,10 +94,42 @@ specify a desired action on Pod object. This is the recommended way for user
 have the infrastructure create Action object for a specific action for a (VM)
 Pod.
 
+Having pods/action subresource allows user/ECS to easily create actions on a
+specific Pod as illustrated in the below reboot example:
+```
+root@fw0000358:~/ActAlk# cat ../YML/reboot.json
+{
+  "apiVersion": "v1",
+  "kind": "CustomAction",
+  "actionName": "Reboot",
+  "rebootParams": {
+    "delayInSeconds": 10
+  }
+}
+
+root@fw0000358:~/ActAlk# curl -X POST http://127.0.0.1:8001/api/v1/namespaces/default/pods/1pod/action -H "Content-Type: application/json" -d @../YML/reboot.json
```

Comment:
this curl call is used at the time kubectl is not supported. now kubectl is updated so we should change to use kubectl create -f action.yaml

Reply:
We don't support create action now. The above curl is supported - which would translate to Action subresource when done programmatically.